plugins/index.md

```diff
@@ -265,6 +266,16 @@ gpgcheck=0
 </pre>
 </div>
 
+<div class="installation_os installation_os_el8">
+<pre>
+[foreman-plugins]
+name=Foreman plugins
+baseurl=https://yum.theforeman.org/plugins/{{page.version}}/el8/x86_64/
+enabled=1
+gpgcheck=0
```

Comment:
Ought we to put in `module_hotfixes=1` like our release RPM does to be on the safe side for plugins to work?

Reply:
We can, yeah